### DISCUSSION TITLE
3.7.7

### DISCUSSION THREAD

**Discussion by tcaiazzi** [Kidel/Kathara#308] [Announcements]
**Bug Fixes**
- Properly set driver options to disable/enable IPV6 (#306)
- Minor adjustments and optimizations

**NOTE**
We encountered some issues while compiling the package on Arch Linux. Our team is actively working on resolving these problems to release the Arch (and AUR) package as soon as possible.

We apologize for any inconvenience this may cause and appreciate your patience.

<hr /><em>This discussion was created from the release <a href='https://github.com/KatharaFramework/Kathara/releases/tag/'>3.7.7</a>.</em>

**Comment by xllauca** [comment]
Hello,
there is an issue when trying to install Kathara with brew on MacOS.

brew install katharaframework/kathara/kathara 
 
==> Downloading https://github.com/KatharaFramework/Kathara/releases/download/3.7.7/Kathara-macos-installer-x86_64-3.7.7.pkg  
curl: (56) The requested URL returned error: 404  
Error: Download failed on Cask 'kathara' with message: Download failed: https://github.com/KatharaFramework/Kathara/releases/download/3.7.7/Kathara-macos-installer-x86_64-3.7.7.pkg


The issue is that in the release assets, the file is named: Kathara-macos-installer-arm64-3.7.7-FIX.pkg. This extra "FIX" causes the installer not to find it. 

Could you please update this, or let me know how I can help?

**Reply by Skazza94** [reply]
Hi @xllauca,
I removed the `-FIX` suffix from the package name. It should work now. 

Cheers,
Mariano
